src/shell/mod.rs

```diff
@@ -363,14 +364,17 @@ impl<'a> Shell<'a> {
 
     /// Evaluates the source init file in the user's home directory.
     pub fn evaluate_init_file(&mut self) {
-        env::home_dir().map_or_else(|| {
-            let stderr = io::stderr();
-            let mut stderr = stderr.lock();
-            let _ = stderr.write_all(b"ion: could not get home directory");
-        }, |mut source_file| {
-            source_file.push(".ionrc");
-            self.execute_script(&source_file);
-        });
+        match get_app_root(AppDataType::UserConfig, &AppInfo{ name : "ion", author : "mmstick" }) {
+            Ok(mut source_file) => {
+                source_file.push("initrc");
+                self.execute_script(&source_file);
+            },
+            Err(_) => { // The error returned could be used for a possibly more descriptive error
+                let stderr = io::stderr();
+                let mut stderr = stderr.lock();
+                let _ = stderr.write_all(b"ion: could not get home directory");
```

Comment:
`eprintln` is in the `1.19.0-beta.2` so presumably it will be part of the official `1.19.0` release.